### PR TITLE
71: Address CRAN review comments.

### LINKED ISSUE #71: Address CRAN review comments
If there are references describing the methods in your package, please add these in the description field of your DESCRIPTION file in the form
authors (year) <doi:...>
authors (year) <arXiv:...>
authors (year, ISBN:...)
or if those are not available: [<https:...>](https://mail.google.com/mail/u/0/...)
with no space after 'doi:', 'arXiv:', 'https:' and angle brackets for auto-linking.
(If you want to add a title as well please put it in quotes: "Title")
- [x] Add references for the R packages we import from

Please add \value to .Rd files regarding exported methods and explain the functions results in the documentation. Please write about the structure of the output (class) and also what the output means. (If a function does not return a value, please document that too, e.g. \value{No return value, called for side effects} or similar)
Missing Rd-tags:
- [x] geom_table.Rd: \value
- [x] stat_n_text.Rd: \value
- [x] stat_prop.Rd: \value
- [x] ttheme_gtdefault.Rd: \value
- [x] ttheme_set.Rd: \value

### PR DESCRIPTION
**Pull request by danielinteractive**
closes #71
[skip vbump]

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -11,7 +11,10 @@ Description: Selected utilities, in particular 'geoms' and 'stats'
     functions, extending the 'ggplot2' package. Note that this package
     does not define the functions itself, but instead imports them from a
     collection of other packages and then exports them. These functions
-    are tested as well to make sure that they work reliably.
+    are tested as well to make sure that they work reliably. Currently,
+    the selected functions are from 'EnvStats' <doi:10.1007/978-1-4614-8456-1>,
+    'GGAlly' <doi:10.5281/zenodo.5009047> and
```
Comment (BFalquet) [inline DESCRIPTION:16]:
```suggestion
    'GGally' <doi:10.5281/zenodo.5009047> and 
```

File: inst/WORDLIST [hunk 2]
```diff
@@ -4,3 +4,9 @@ Sabanés
 funder
 geoms
 ggplot
+doi
+EnvStats
+GGAlly
```
Comment (BFalquet) [inline inst/WORDLIST:9]:
```suggestion
GGally
```

I think that it is `GGally` https://www.rdocumentation.org/packages/GGally/versions/1.5.0 or are you referring to another package ?

**Review by BFalquet — APPROVED** [review]
Small comment inside about the name of the library. Please confirm before merging. Other thank that, look good. Thank you.

File: DESCRIPTION (hunk 1 above)
Comment (danielinteractive) [reply DESCRIPTION:16]: good catch thanks!

**Pull request merged**